migrations/2022-11-13-181529_create_taglines/up.sql

```diff
@@ -0,0 +1,7 @@
+create table tagline (
+  id serial primary key,
+  local_site_id int references local_site on update cascade on delete cascade not null,
```

Comment:
There is no reason to include the site id here because these taglines dont federate. All taglines stored in the database are for the local site.

I commented this before, not sure where the comment ended up.

Reply:
In the comments above, dessalines responded to your previous comment:

"Yes, mainly because its not good to have unconnected, "dangling" data. We can't anticipate the data transformations and joins we might need. Plus both are logically local site settings."

Reply:
Alright.